### PR TITLE
COMMERCE-4659 use proper inventory audit type

### PR DESCRIPTION
**Pull request by luca-pellizzon**

### CODE REVIEW

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -1046,15 +1046,15 @@ payments=Payments
...
 paypal-subtotal-discount=Subtotal Discount
-paypal-subtotal-discount-description=Applied Discount to Subtotal
-paypal-taxes=Applied Taxes
-paypal-taxes-description=Taxes
+paypal-subtotal-discount-description=Applied Discount to Subtotal.
+paypal-taxes=Taxes
+paypal-taxes-description=Applied taxes.
 paypal-total-discount=Total Discount
-paypal-total-discount-description=Applied Discount to Total
+paypal-total-discount-description=Applied Discount to Total.
```
Comment (dsitu) [inline modules/apps/commerce/commerce-lang/src/main/resources/content/Language.properties:1057]:
```suggestion
paypal-total-discount-description=Applied discount to the total.
```

File: modules/apps/commerce/commerce-lang/src/main/resources/content/Language.properties [hunk 2]
```diff
@@ -1602,24 +1602,25 @@ workflow-status=Workflow-status
 x-available={0} Available
 x-base-price-list={0} Base Price List
 x-base-promotion={0} Base Promotion
-x-created-a-transfer-from-x-to-x={0} created a transfer from {1} to {2}
-x-created-shipment-x-consuming-quantity={0} created Shipment {1}, consuming quantity
+x-created-a-transfer-from-x-to-x={0} created a transfer from {1} to {2}.
+x-created-shipment-x-consuming-quantity={0} created Shipment {1}, consuming quantity.
```
Comment (dsitu) [inline modules/apps/commerce/commerce-lang/src/main/resources/content/Language.properties:1606]:
```suggestion
x-created-shipment-x-consuming-quantity={0} created shipment {1}, consuming quantity.
```

File: modules/apps/commerce/commerce-lang/src/main/resources/content/Language.properties [hunk 3]
```diff
@@ -1602,24 +1602,25 @@ workflow-status=Workflow-status
...
 x-is-not-available={0} is not available.
 x-notes={0} Notes
 x-of-x-products-selected={0} of {1} Products Selected
-x-placed-order-x-booking-quantity={0} placed order {1}, booking quantity
+x-placed-order-x-booking-quantity={0} placed order {1}, booking quantity.
 x-products-available={0} Products Available
-x-restocked-the-quantity-from-shipment-x={0} restocked the quantity from Shipment {1}
-x-updated-the-quantity-on-hand={0} updated the quantity on hand
-yes-restock-the-items=Yes, restock the items
+x-restocked-the-quantity-from-shipment-x={0} restocked the quantity from Shipment {1}.
```
Comment (dsitu) [inline modules/apps/commerce/commerce-lang/src/main/resources/content/Language.properties:1614]:
```suggestion
x-restocked-the-quantity-from-shipment-x={0} restocked the quantity from shipment {1}.
```

File: modules/apps/commerce/commerce-lang/src/main/resources/content/Language.properties [hunk 4]
```diff
@@ -1046,15 +1046,15 @@ payments=Payments
 paypal-configuration-help=Please go to the {0}PayPal Developer Dashboard{1} to create a new PayPal REST API application, then copy the client ID and client secret in the fields below.
 paypal-description=Pay via PayPal.
 paypal-shipping=Shipping Costs
-paypal-shipping-description=Costs of shipping
+paypal-shipping-description=Costs of shipping.
```
Comment (dsitu) [inline modules/apps/commerce/commerce-lang/src/main/resources/content/Language.properties:1049]:
```suggestion
paypal-shipping-description=Cost of shipping.
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
those tests are not failing in PR
https://github.com/liferay-commerce/liferay-portal/pull/177
this one should not prevent them to pass.
Don't know why they failed.

**Comment by igorbeslic** [comment]
fixed in https://github.com/brianchandotcom/liferay-portal/pull/94157

**Comment by dsitu** [comment]
Closing sent directly here https://github.com/brianchandotcom/liferay-portal/pull/94154.

**Pull request closed without merging**